### ISSUE TITLE
UI issue on ipad/safari

### ISSUE THREAD

**Issue by goyatz** [bjpirt/apps.mirobot.io#1]
- cant Chance variables and Numbers or Selections (fortward, backward) 
- Do not run the loop Feature

**Comment by bjpirt** [comment]
@broo0ose please could you try this again now? There was a recent fix that should sort this out.

**Comment by broo0ose** [comment]
No sorry, it seems the same on Android.

Looks like HTML5 drag is not supported on Android or IOS
http://caniuse.com/#search=drag

Bruce

**Comment by bjpirt** [comment]
True, although blockly does work on tablets. I tried this page on my iOS device and it worked fine:

https://blockly-games.appspot.com/turtle

I'll dig into it shortly.

**Comment by broo0ose** [comment]
and this Blockly page says it supports Android. I didn't realise it was a Google production. 
https://developers.google.com/blockly/

**Comment by bjpirt** [comment]
The original issue was that loops weren't working - @goyatz could you please test whether this is still the case?

Now it also seems that drag and drop isn't working on touch browsers (although it did originally seem to be OK), which I'll take a look at today.

**Comment by bjpirt** [comment]
Yup, the file save update broke touch devices - have fixed this and will deploy shortly.

**Comment by goyatz** [comment]
Good Morning, Loops work Fine, i cant change any value in Loops Or directions ... BR uwe

**Comment by bjpirt** [comment]
Hmm, odd. Will reopen and take another look.
